src/EditorFeatures/VisualBasicTest/Diagnostics/AddImport/AddImportTests.vb

```diff
@@ -75,7 +76,7 @@ Namespace SomeNamespace
 End Namespace")
         End Function
 
-        <Fact, Trait(Traits.Feature, Traits.Features.CodeActionsAddImport)>
+        <Fact(Skip:="https://github.com/dotnet/roslyn/issues/41484"), Trait(Traits.Feature, Traits.Features.CodeActionsAddImport)>
```

Comment:
This test was passing as the test framework was never performing the validation for the passed in expected `priority` argument.